packages/a11y-testing/src/definitions/Popup/popupBehaviorDefinition.ts

```diff
@@ -0,0 +1,70 @@
+import { Rule } from './../../types';
+import { BehaviorRule } from './../../rules/rules';
+
+export const popupBehaviorDefinitionTriggerSlotNotTabbable: Rule[] = [
+  BehaviorRule.slot('trigger')
+    .forProps({ tabbableTrigger: true })
+    .hasAttribute('tabIndex', '0')
+    .hasAttribute('aria-haspopup', 'dialog')
+    .description(`if 'tabbableTrigger' is 'true'.`),
+  BehaviorRule.slot('trigger')
+    .forProps({ tabbableTrigger: true })
+    .hasAttribute('tabIndex', '0')
+    .description(`if it is 'not tabbable' and 'tabbableTrigger' is 'true'.`),
+  BehaviorRule.slot('trigger')
+    .forProps({ tabbableTrigger: false })
+    .doesNotHaveAttribute('tabIndex')
+    .description(`if 'tabbableTrigger' is 'false'.`),
+  BehaviorRule.slot('trigger')
+    .forProps({ tabbableTrigger: false })
+    .doesNotHaveAttribute('aria-haspopup')
+    .description(`if 'tabbableTrigger' is 'false'.`),
+];
+
+export const popupBehaviorDefinitionTriggerSlotTabbable: Rule[] = [
```

Comment:
I expect that we will have a single definition for the matching behavior. Why we have 5 exports there? 🤔

Reply:
This appears like we have two slots which we need to test like `trigger` and `popup`. 
And then `trigger` is special in the way, that behaves differently when we receive: 
- Tabbable element  
- not tabbable element 
- element with tabIndex already set... 

So in the end I ended with 4 definitions which using in the tests. 

For purpose of doc generation I still exporting only one "popupBehaviorDefinition".